fabric/bin/peer.sh

```diff
@@ -35,6 +37,43 @@ export LD_LIBRARY_PATH=${LD_LIBRARY_PATH:+"${LD_LIBRARY_PATH}:"}${FPC_TOP_DIR}/t
 # Lifecycle Chaincode command wrappers
 #--------------------------------------------
 
+handle_lifecycle_ercc_package() {
+    # check required parameters
+    [ ! -z "${ERCC_PACKAGE}" ]      || die "undefined ercc package"
+    [ ! -z "${ERCC_LANG}" ]         || die "undefined ercc lang"
```

Comment:
ok - I don't have a strong opinion here. But it seems calling it ERCC_TYPE rather than ERCC_LANG makes your code much better to understand. Ask me in two month what ERCC_LANG is ... and I will answer Go!

Reply:
I agree that "type" is more appropriate than "lang"... changed!